docs/extensions/shoot-maintenance.md

```diff
@@ -0,0 +1,10 @@
+# Shoot maintenance
+Shoots configure a maintenance time window in which Gardener performs maintenance related tasks like infrastructure reconciliation, updating K8s or machine image versions.
+
+## Restart control plane controllers
+It is possible to configure Gardener to restart/delete certain control plane pods during a shoot's maintenance.
```

Comment:
```suggestion
As an extension controller it is possible to make Gardener restarting/deleting certain control plane pods during a shoot's maintenance.
```

Reply:
Actually the operator configures if the pods are restarted. I changed the text a bit to make it more clear.